source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnView.mm

```diff
@@ -18,10 +18,13 @@ - (void)config:(nullable NSDictionary<NSString *, id> *)attributes
 - (void)addArrangedSubview:(UIView *)view
 {
     // if auto, maintain content size whenever possible
-    if([self.columnWidth isEqualToString:@"auto"]){
-        [view setContentHuggingPriority:UILayoutPriorityDefaultHigh forAxis:UILayoutConstraintAxisHorizontal];
-        [view setContentCompressionResistancePriority:UILayoutPriorityRequired forAxis:UILayoutConstraintAxisHorizontal];
+    if([self.columnWidth isEqualToString:@"auto"]){        
+        [view setContentHuggingPriority:251 forAxis:UILayoutConstraintAxisHorizontal];
```

Comment:
>251 [](start = 40, length = 3)

Should these magic priority numbers be `#define`d somewhere? Maybe an enum or something?

Reply:
sounds good

---
In reply to: [303630153](https://github.com/microsoft/AdaptiveCards/pull/3195#discussion_r303630153) [](ancestors = 303630153)